tests/travis_run_tests

```diff
@@ -5,8 +5,9 @@
 
 set -ev
 
-blt validate:all --no-interaction
+blt blt:telemetry:disable --no-interaction
```

Comment:
I wonder if we could switch back to using the default Travis script included with BLT. I'm curious if BLT 12 resolved the odd timeouts we were running into before.

Reply:
Now would be a good time to try!

Reply:
I'm not sure exactly what this would entail. Can you provide instructions? Or would you want to try pushing an update to the branch?

Reply:
This seems to work. It doesn't stream the output which is intentional for reasons I don't quite understand. I'll remove our version of the script. If it starts timing out in the future, we can revert.